### DISCUSSION TITLE
if i use i3 6000  (low speed ) with 3060  gpu  ( normal gpu ) what will happen ??

### DISCUSSION THREAD

**Discussion by maloadjav** [openai/whisper#741] [Q&A]
do i need to upgrade both cpu +gpu if i want to use this program
or i just need to buy  12 gig gpu  (3060 )  and it will work  too .

**Comment by krylm** [comment]
I am using whisper (even large size model) with Ryzen 5900x & MSI Ventus 3x 3060 12 GB gpu without any problems.

**Reply by maloadjav** [reply]
oh. thank you so much.
so you think that is not much about cpu right ?
my cpu is very old for these day but i dont want to upgrade it if its not that need

**Reply by MiscellaneousStuff** [reply]
You won't need to upgrade your CPU to use this module. The CPU is only used for preprocessing the audio input and sending the data to your CPU so a much better CPU would make a minor difference.

**Reply by maloadjav** [reply]
okay. thank you so much for explaining
